pkg/controller/pxc/status.go

```diff
@@ -103,7 +104,23 @@ func (r *ReconcilePerconaXtraDBCluster) updateStatus(cr *api.PerconaXtraDBCluste
 		}
 
 		cr.Status.ProxySQL = proxyStatus
-		cr.Status.Host = cr.Name + "-" + "proxysql"
+
+		cr.Status.Host = cr.Name + "-" + "proxysql." + cr.Namespace
+		if cr.Spec.ProxySQL != nil && cr.Spec.ProxySQL.ServiceType != nil && *cr.Spec.ProxySQL.ServiceType == corev1.ServiceTypeLoadBalancer {
+			svc := &corev1.Service{}
+			err := r.client.Get(context.TODO(),
+				types.NamespacedName{
+					Namespace: cr.Namespace,
+					Name:      cr.Name + "-" + "proxysql",
+				}, svc)
+			if err != nil {
+				return errors.Wrap(err, "get proxysql service")
+			}
+			for _, i := range svc.Status.LoadBalancer.Ingress {
+				cr.Status.Host = i.IP
```

Comment:
sometimes it is IP, sometimes it is Hostname, also it takes time to get it from Ingress
please take function from PSMDB Operator https://github.com/percona/percona-server-mongodb-operator/blob/master/pkg/psmdb/service.go#L148-L185

Reply:
added https://github.com/percona/percona-xtradb-cluster-operator/pull/373/commits/a3672fcc146c5e81a5adabd0d109b43771dfe922

Reply:
But there is still no workaround for ingress lags, innit?